src/main/webapp/style.css

```diff
@@ -1,7 +1,10 @@
 /* CSS Document */
 @import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
 body {
-	font-family: 'Roboto', sans-serif;
+    font-family: 'Roboto', sans-serif;
+    background-image: url(blue_gradian.png);
+    background-repeat: repeat-x;
+border-style: hidden;    

```

Comment:
This breaks on the page where we display the list of images like so: 
![image](https://user-images.githubusercontent.com/11355355/115074054-2b799180-9ec7-11eb-9ef2-3661c8f5d0af.png)

Reply:
That's a good point. Likely because it's set to repeat-x